recipe/meta.yaml

```
@@ -392,6 +437,7 @@ outputs:
             "test/test_nn.py",
             "test/test_torch.py",
             "test/test_xnnpack_integration.py",
+            "test/inductor/test_torchinductor.py",                                      # [cuda_compiler_version != "None"]
             "-m \"not hypothesis\"",
         ]) %}
```

Unless something changed recently, it's not possible to use selectors in jinja expressions. But it's possible to do
```suggestion
            "test/test_xnnpack_integration.py",
        ] + (cuda_compiler_version != "None") * ["test/inductor/test_torchinductor.py"]) %}
```
(and then either move `-m \"not hypothesis\"` to the front of the list - or even better, to the pytest invocation).